pdufour/postcard-create
# feat(base): Add initial postcard creation app

# what
add angular bootcamp code so that you can create a new postcard

# testing instructions
- run the server
- navigate to http://localhost:8888/#!/create and see that you can create a new postcard
